src/modules/rover_ackermann/module.yaml

```diff
@@ -175,3 +190,25 @@ parameters:
         increment: 0.01
         decimal: 2
         default: -1
+
+      RA_MAX_ACCEL:
+        description:
+          short: Maximum acceleration for the rover
+        type: float
+        unit: m/s^2
+        min: -1
+        max: 100
+        increment: 0.01
+        decimal: 2
+        default: -1
+
+      RA_MAX_STR_RATE:
```

Comment:
We cannot use the already existing CA_SV${i}_SLEW and CA_R${i}_SLEW because we don't use the control allocation module?

Reply:
Yes, these parameters don"t seem to exist if you don't use it. I also prefer having all ackermann related parameters in one place, but I could change it if necessary.

Reply:
Let's keep it in the back of our minds, but it's not clear thing which is better or worse.